ui/tests/acceptance/oidc-provider-test.js

```diff
@@ -218,10 +210,7 @@ module('Acceptance | oidc provider', function (hooks) {
       currentURL().startsWith('/vault/auth'),
       'Does not redirect to auth because user is already logged in'
     );
+    await waitFor('[data-test-consent-form]');
     assert.dom('[data-test-consent-form]').exists('Consent form exists');
-
-    //* clean up test state
```

Comment:
I'm a little suspicious that we no longer have to clear these records, but I guess it's good if we no longer need to?

Reply:
Let me add them back in, just in case.